fastlane_bot/config/network.py

```diff
@@ -237,7 +236,7 @@ class ConfigNetwork(ConfigBase):
 
     GAS_ORACLE_ADDRESS = None
 
-    CARBON_V1_FORKS = [CARBON_V1_NAME, CARBON_V1_X2_NAME]
+    CARBON_V1_FORKS = []
```

Comment:
Not clear why this is needed here (why not just create it in the class's post-init function?)